[2022-12-03 15:01:27,403] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T19:01:00+00:00 [queued]>
[2022-12-03 15:01:27,782] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T19:01:00+00:00 [queued]>
[2022-12-03 15:01:27,783] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 15:01:27,784] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-03 15:01:27,784] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 15:01:28,229] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 19:01:00+00:00
[2022-12-03 15:01:28,331] {standard_task_runner.py:52} INFO - Started process 902 to run task
[2022-12-03 15:01:28,433] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T19:01:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpz3tl8li1', '--error-file', '/tmp/tmpe3msf7td']
[2022-12-03 15:01:28,463] {standard_task_runner.py:80} INFO - Job 254: Subtask get_last_id
[2022-12-03 15:01:30,979] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T19:01:00+00:00 [running]> on host b41792e8e843
[2022-12-03 15:01:35,127] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T19:01:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T19:01:00+00:00
[2022-12-03 15:01:35,789] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-03 15:01:36,442] {python.py:173} INFO - Done. Returned value was: None
[2022-12-03 15:01:37,775] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T190100, start_date=20221203T150127, end_date=20221203T150137
[2022-12-03 15:01:38,502] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-03 15:01:40,033] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
